regression-test/data/nereids_p0/sql_functions/datetime_functions/test_date_function.out

```diff
@@ -313,7 +313,7 @@ February
 749027
 
 -- !sql --
-1196389819
+1.196389819E9
```

Comment:
the result is not in the right format. what if just run this SQL (not via regression Groovy framework)? maybe use decimalV2 could be better